jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -516,12 +523,23 @@ YUI.add('juju-env-go', function(Y) {
       environment definition.
 
       @method environmentGet
+      @param {Function} callback A callable that must be called once the
+        operation is performed. It will receive an object with an "err"
+        attribute containing a string describing the problem (if an error
+        occurred), or with the "config" attribute if everything went well.
+      @return {undefined} Sends a message to the server only.
     */
-    environmentGet: function() {
+    environmentGet: function(callback) {
```

Comment:
I actually prefer the single callback using the nodejs style where the first argument is an error `callback(err, data)` :) but for now following the same pattern is good